t/plugin/google-cloud-logging.t

```diff
@@ -31,6 +31,43 @@ add_block_preprocessor(sub {
     if (!defined $block->request) {
         $block->set_value("request", "GET /t");
     }
+     my $http_config = $block->http_config // <<_EOC_;
+server {
+        listen 12001;
+
+        location /google-cloud-logging/test {
+            content_by_lua_block {
+                ngx.req.read_body()
+                local data = ngx.req.get_body_data()
+                ngx.log(ngx.WARN, "google cloud logging request body: ", data)
+                ngx.log(ngx.ERR, data)
+                ngx.say('test-http-logger-response')
+                ngx.log(ngx.WARN, "loggingBody:", data)
```

Comment:
The logging of the data is a bit confusing. The same data here is logged by three times.